proto/regen/ecocredit/marketplace/v1/tx.proto

```diff
@@ -25,6 +25,9 @@ service Msg {
   // Buy creates credit buy orders.
   rpc Buy(MsgBuy) returns (MsgBuyResponse);
```

Comment:
Since this isn't support it yet, we should just delete `Buy` and add it back later

Reply:
deleted